Changelog.md

```diff
@@ -1,3 +1,9 @@
+## 4.2.1 - 2023-09-21
+
+### Added
+
+-   Bundle Quickstart app with the launcher installer
```

Comment:
Suggestion: “Added Quickstart app to easily get going with an nRF9160 or nRF9161 DK.”

I think this makes it easier for our users to understand what the change means for them (and it adds the `.` we have at the end of each entry).

Reply:
After todays decision “Quickstart” needs to be changed to “Quick Start”.

Reply:
Applied